src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.scss

```diff
@@ -2,7 +2,7 @@
 
 .grid {
   display: grid;
-  grid-template-columns: 11ch [submission-name] auto [submission-details] 1fr [control-buttons] 20%;
+  grid-template-columns: 11ch [submission-name] auto [submission-details] 1fr [controls-section] 20%;
```

Comment:
Just curious why this is `controls-section` now?

Reply:
🤷‍♂️  It contains more than buttons.